### PR TITLE
Adding terms for LGBT and suicide prevention issues.

### PR DESCRIPTION
**Pull request by capitocapito**
Added terms, sources, and notes, and added to the data variable in extract.js.

My first pull request. :) 

I was less familiar with node that I thought... I have node installed, but I wasn't certain how to test a local copy of the stuff I had forked. I went straight for the .yml files, but I'm not sure how to verify that this is all good.

Let me know if there are any other steps I should take. Cheers!

### CODE REVIEW

**Comment by wooorm** [comment]
@capitocapito Thanks a lot!

First: very nice work, well done, the patterns are great additions.
Second: some slight points.

*   Could you add a final new line at the end of files? Google them for more
    info :)

*   Maybe the AMAB/DMAB/AMAF/DMAF pattern should be of type `simple`,
    right now, there are is a warning no `born a man or born a female`,
    just for `born a man` or `born a female`

*   The alternative for `bathroom bill` is pretty hefty:
    `non-discrimination law/ordinance`. Could you replace it with two
    seperate suggestions? `non-discrimination law` and
    `non-discrimination ordinance` is fine.

*   You can build the project using `npm run build`, and ensure everything works
    using `npm run make`; Before these two you need to `npm install` first
    though.

And, when you’re done, could you rebase the branch? Google can also help you there re how to do that :smile:

**Comment by capitocapito** [comment]
@wooorm Gotcha.

- Will add new lines.

- Understood. I may have misinterpreted how the or works (I was thinking it was necessary for categories). I'll change that and test.

- Will split the bathroom bill suggestions.

- Thanks for the hints! I'll play around more with npm, and rebasing shouldn't be a problem. 

Thanks for the help! :D

**Comment by wooorm** [comment]
:+1:

**Comment by wooorm** [comment]
@capitocapito How are things going with this? I really appreciate your previous work :ok_hand:

File: script/lgbtq.yml [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  considerate:
+    - cross-dresser
+  inconsiderate:
+    - transvestite
+- type: simple
+  source: http://www.glaad.org/reference/transgender
+  considerate:
+    - transition
+  inconsiderate:
+    - sex-change
```
Comment (danielhickman) [inline script/lgbtq.yml]: What about "the op" or "post op"?

Comment (wooorm) [inline script/lgbtq.yml]: Could you provide examples of where theses cases might be considered insensitive?

Comment (danielhickman) [inline script/lgbtq.yml]: @wooorm I'm not sure if it is but I thought it might be a good idea to bring it up.

**Comment by capitocapito** [comment]
@wooorm Hello! Busy week.

I've been able to complete the tasks that we outlined above, but I still haven't been able to test. I have node installed on my end, and I was able to install, build, and make retext-equality but I'm not sure how to test it locally with my copy of alex. So far, I've tried adding the local copy of retext-equality as a dependency for alex, but after doing the same install, build, and run, when I pass anything, new or not, (e.g. `echo 'retard' | node alex`) I am simply presented with a new command line with no output.  

I'll push my fixes on retext-equality, though. I'd hate to make this a personal troubleshooting thread. :P

File: script/lgbtq.yml (hunk 1 above)
Comment (capitocapito) [inline script/lgbtq.yml]: That's actually a note that was included in one of the articles: 

>Referring to a "sex-change operation," or using terms such as "pre-operative" or "post-operative," inaccurately suggests that one must have surgery in order to transition.

**Comment by wooorm** [comment]
@capitocapito Cool! In theory, when you have retext-equality working (with some tests), you could run `npm link` from inside that package.
Then, you can `git clone` alex, install its dependencies, and run `npm link retext-equality` in it.

Both these actions set-up some sort of symlink between the two, enabling you to test your changes in retext-equality by running, e.g., `alex <<< "foo bar retard baz"`

See this npm doc for more info: https://docs.npmjs.com/cli/link

**Comment by wooorm** [comment]
@capitocapito Are you still working on this? If not, I’d love to try my hands on this and finish the PR (If you’d like, it’s possible to still keep you as an author in Git, which will show your avatar on GitHub).

**Comment by wooorm** [comment]
@capitocapito ping!

**Comment by capitocapito** [comment]
Hello! I haven't had a chance to work on this at all. I would say just go with the updates that the other person fixed with the spacing, since that's all I had done at this point and they had committed it first. Thank you, though!

**Pull request closed without merging**